src/sparseml/export/export.py

```diff
@@ -112,6 +117,7 @@ def export(
 
     # choose the appropriate device
     device = default_device() if device == "auto" else device
+    device = use_single_gpu(device) if "cuda" in device else device
```

Comment:
We want to be in full control over the placement of the model on the device.
User can either specify the `device:str` ourselves (to `cpu`, `cuda`, `cuda:0`) or keep the default `auto` configuration. If `auto`, we use `default_device()` helper function that returns a proper device string.
Question 1: shouldn't we default to `cpu` here?
Question 2: if requested, should we allow the user to place the model over multiple CUDA devices?

Added the `use_single_gpu` function